### PR TITLE
Added IP logging

### PR DESCRIPTION
**Pull request by homerr**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-duckdns/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Added logging to the `duck.log` file to include the discovered IP address(es) 

## Benefits of this PR and context:

Closes issue #72 feature request 

## How Has This Been Tested?

Tested locally with success;

``` homer@homer-MacBook-Pro  ~/docker/ddns  cat duck/duck.log
Detecting IPv4 via CloudFlare
Your IP was updated at Fri Mar 29 09:26:54 UTC 2024 to IPv4: 82.44.x.x
```

### CODE REVIEW

**Review by homerr — COMMENTED** [review]
All changes are good!

**Comment by homerr** [comment]
@aptalca  should be good to merge now if I could have your approval :)

**Review by aptalca — APPROVED** [review]

**Pull request merged**
